lms/models/lti_event.py

```diff
@@ -0,0 +1,94 @@
+from enum import Enum
+from functools import lru_cache
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.mutable import MutableDict
+from sqlalchemy import insert
+
+from lms.db import BASE
+
+
+class LTIEventType(BASE):
```

Comment:
Following the roles table, saving a bit of data for type in the main events table. Not that big but it will add up.